### PR TITLE
add partner name in managementpartner commands response

### PR DESCRIPTION
**Pull request by jeffrey-ACE-zz**
## Checklist

- [x] I have read the [_Submitting Changes_](https://github.com/Azure/azure-powershell/blob/preview/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](https://github.com/Azure/azure-powershell/blob/preview/CONTRIBUTING.md)
- [ ] The title of the PR is clear and informative
- [x] The appropriate [change log has been updated](https://github.com/Azure/azure-powershell/blob/preview/CONTRIBUTING.md#updating-the-change-log)
- [x] The PR does not introduce [breaking changes](https://github.com/Azure/azure-powershell/blob/preview/documentation/breaking-changes/breaking-changes-definition.md)
- [x] If applicable, the changes made in the PR have proper test coverage
- [x] For public API changes to cmdlets:
    - [x] the changes have gone through a [cmdlet design review](https://github.com/Azure/azure-powershell-cmdlet-review-pr)
    - [x] the cmdlet markdown files were [generated using the `platyPS` module](https://github.com/Azure/azure-powershell/blob/preview/documentation/development-docs/help-generation.md)

### CODE REVIEW

**Comment by jeffrey-ACE** [comment]
@cormacpayne , added the release information and resolved the conflict too

**Comment by jeffrey-ACE** [comment]
@azuresdkci Retest this please

File: src/ResourceManager/ManagementPartner/ChangeLog.md [hunk 1]
```diff
@@ -19,6 +19,8 @@
 -->
 ## Current Release
 
-## Version 0.1.0
-* First release of Management Partners cmdlets
+## Version 0.1.1
```
Comment (cormacpayne) [inline src/ResourceManager/ManagementPartner/ChangeLog.md]: @jeffrey-ACE one small comment -- you don't need to add a new header, you can put the text you're adding under the `## Current Release` header

**Comment by jeffrey-ACE** [comment]
@cormacpayne can you please approve and merge this PR.

**Pull request merged**